### PR TITLE
Bump rubygem-smart_proxy_discovery to 1.0.5-10

### PR DESCRIPTION
**Pull request by nofaralfasi**

### CODE REVIEW

File: packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec [hunk 1]
```diff
@@ -109,6 +81,9 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery.yml.example \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Mar 07 2024 nofaralfasi <nalfassi@redhat.com> 1.0.5-1
+- Update to 1.0.5-1
```
Comment (nofaralfasi) [inline packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec:85]: @evgeni  Any idea why it didn't jump to `1.0.5-10`?

Comment (nofaralfasi) [reply packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec:85]: And what about line 5? `%global foreman_proxy_min_version 1.24` is correct? (used to be 1.25)

Comment (evgeni) [reply packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec:85]: That comes from https://github.com/theforeman/foreman-packaging/blob/7511869cc09cf038e6aaa503a08b83412c32f46a/gem2rpm/smart_proxy_plugin.spec.erb#L15, no idea which version your plugin needs :)

File: packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec [hunk 2]
```diff
@@ -109,6 +81,9 @@ mv %{buildroot}%{gem_instdir}/settings.d/discovery.yml.example \
 %doc %{gem_instdir}/README.md
 
 %changelog
+* Thu Mar 07 2024 nofaralfasi <nalfassi@redhat.com> 1.0.5-10
+- Update to 1.0.5-10
```
Comment (evgeni) [inline packages/plugins/rubygem-smart_proxy_discovery/rubygem-smart_proxy_discovery.spec:85]:
```suggestion
- Regenerate spec file based on the latest template
```

(or something along these lines)

same for the commit message, then we're good to go :)

**Review by evgeni — APPROVED** [review]

**Pull request merged**